src/server/src/routes/job.ts

```diff
@@ -9,6 +9,8 @@ import {
     getJobs,
 } from '@modules/job';
 import logger from '@shared/Logger';
+import { parse } from 'path';
```

Comment:
What are these imports, parse and start, for?

Reply:
I'm not sure. It was there when I gitpulled

Reply:
I don't think we need it. Can you remove it?

Reply:
removed